### DISCUSSION TITLE
Print data size function

### DISCUSSION THREAD

**Discussion by James-MF-Thomas** [UltimaHoarder/UltimaScraper#617] [Q&A] [answered]
Don't know how difficult it would be to get a print function of each api's total size in gb/mb scrapped from each model, kinda like a checksum. 

nothing fancy just below or on the same line where it prints  "Scrape Completed:"  

Finished processing post
Scrape Completed (2.4Gb)

**Comment by UltimaHoarder** [comment, marked as answer]
Impossible. OnlyFans doesn't keep file size in their api anymore.

**Comment by James-MF-Thomas** [comment]
fair enough
